certified-connectors/Zuva DocAI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,837 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Zuva DocAI",
+    "description": "Embed Contracts AI into your product workflows using Zuva DocAI. No need for AI development from the ground up, leverage the DocAI APIs to extract information from your contracts.",
+    "contact": {
+      "name": "Zuva Support",
+      "url": "https://zuva.ai/support/",
+      "email": "support@zuva.ai"
+    }
+  },
+  "host": "us.app.zuva.ai",
```

Comment:
You probably already looked into this, but just want to confirm: is it not possible to pass the host as a parameter, similar to the token?

If that is the case, I assume if we wanted to add the other regions in the future we would have to create separate connectors? Perhaps we consider updating the title to include the region, something like `"Zuva DocAI (US)"`?

Reply:
I did look into it and couldn't find a good way, but your rephrasing of the question led me to try again and I found the way to do it properly: https://docs.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl

Reply:
That's great, nice find!